packages/patternfly-4/content/design-guidelines/content/grammarandterminology.md

```diff
@@ -73,6 +64,63 @@ Use sentence-style capitalization for all text elements, **except:**
 
 Sentence style capitalizes only the first word of each sentence and proper nouns, like names.
 
+## Date and time
+
+Whenever possible, it is best to enable users to customize the format in which date and time are displayed in the interface.
+
+These date and time standards are based in American English, but date and time is formatted differently depending on language.
+
+### Date
+Use the default US style of **Month DD, YYYY**. Use **DD Month YYYY** for non-US locations.
+
+Whenever possible, write out the full name of the month. If you have space constraints, use the 3 letter abbreviation for the name of the month.
+
+Jan
+Feb
+Mar
+Apr
+May
+Jun
+Jul
+Aug
+Sep
+Oct
+Nov
+Dec
+
+**Examples**
+Thursday, 01 January 2019
+07 January 2019
+07 Jan 2019
+
+When you represent the date as a numeric value or label, use the ISO Standard 8601: YYYY-MM-DD.
+
+### Time
+By default, use the 12 hour clock followed by AM or PM. Enter a space after the last number to separate the time from the abbreviation.
+
+When representing date and time, include your time stamp after the date and separate with a comma.
+
+**Examples**
+Thursday, 21 January 2019, 9:38:11 PM
+9:38:11 PM
+9:38 PM
+
+**Absolute and relative time**
+The use of absolute or relative times stamps will depend on your user needs and use case. If your user is primarily interested in understanding the exact date and time that an event occurred, use an absolute timestamp.
+
+If users are interested in how long ago an event occurred, use a relative timestamp.
+
+For consistent timestamp formatting, we recommend using the following examples:
+
+| Timeframe  | Usage |
```

Comment:
I'm not exactly sure what this is recommending. Are you saying that for an event that occurred within the past hour, I would use relative time, but then switch to absolute time after an hour passes? I agree that this is a tough call regarding the right approach.

Reply:
I'm plugging my modification to this relative time here! It is more similar to [momentjs](https://momentjs.com/)
https://docs.google.com/document/d/1hivSfe48717VhlLRtjyqReS5twbSLuPC6MISJt3MTG4/edit?usp=sharing

![Xnip2019-03-28_15-52-48](https://user-images.githubusercontent.com/23172038/55188348-91284300-5171-11e9-835f-c1d530c563b2.jpg)